doc/nrf/ug_nrf91_features.rst

```diff
@@ -160,70 +160,90 @@ The :ref:`nrfxlib:nrf_modem` is released as an OS-independent binary library in
 The Modem library integration layer fulfills the integration requirements of the Modem library in |NCS|.
 For more information on the integration, see :ref:`nrf_modem_lib_readme`.
 
+.. _nrf9160_fota:
 
-.. _nrf9160_ug_band_lock:
+FOTA upgrades
+=============
 
-Band lock
-*********
+|fota_upgrades_def|
+FOTA upgrades can be used to apply delta patches to the :ref:`lte_modem` firmware, full :ref:`lte_modem` firmware upgrades, and to replace the upgradable bootloader or the application.
 
-The band lock is a functionality of the application that lets you send an AT command to the modem instructing it to operate only on specific bands.
-The band lock is handled by the LTE Link Control driver.
-By default, the functionality is disabled in the driver's Kconfig file.
+.. note::
+   Even though the Secure Partition Manager and the application are two individually compiled components, they are treated as a single binary blob in the context of firmware upgrades.
+   Any reference to the application in this section is meant to indicate the application including the Secure Partition Manager.
 
-The modem can operate in the following E-UTRA Bands: 1, 2, 3, 4, 5, 8, 12, 13, 17, 18, 19, 20, 25, 26, 28, and 66.
-To check which bands are supported for a particular modem firmware version, see the `nRF9160 product website (compatible downloads)`_.
+To perform a FOTA upgrade, complete the following steps:
 
-You can use the band lock to restrict modem operation to a subset of the supported bands, which might improve the performance of your application.
-To check which bands are certified in your region, visit `nRF9160 Certifications`_.
+1. Make sure that your application supports FOTA upgrades.
+      To download and apply FOTA upgrades, your application must use the :ref:`lib_fota_download` library.
```

Comment:
If you intend to obtain FOTA files from nRF Cloud, see additional requirements in :ref:\`lib_nrf_cloud_fota\`.